packages/angular/src/generators/setup-mf/files/webpack/module-federation.config.js__tmpl__

```diff
@@ -2,7 +2,7 @@ module.exports = {
   name: '<%= name %>',<% if(type === 'host') { %>
   remotes: [<% remotes.forEach(function(remote) { %>'<%= remote.remoteName %>',<% }); %>]<% } %><% if(type === 'remote') { %>
   exposes: {<% if(standalone) { %>
-    './Routes': '<%= projectRoot %>/src/app/remote-entry/routes.ts',<% } else { %>
+    './Routes': '<%= projectRoot %>/src/app/app.routes.ts',<% } else { %>
```

Comment:
I guess here, either would work. It _is_ probably better to stick to just the remote-entry stuff.

Reply:
Thinking more about this, i think it should stay as is, because standalone remotes will not have a `entry.routes.ts` file.

It uses `app.routes.ts` and by leaving it like this, it stays consistent across both those scenarios.

Reply:
Then, why do we generate an `entry.routes.ts`? It doesn't make sense to duplicate the routing configuration if we're only going to expose the outer `app.routes.ts`.

That said, I don't agree on using/exposing the outer `app.routes.ts`. The whole `app.module.ts` (where and why `app.routes.ts` is used) in a remote app is only there to be able to serve the remote on its own (which is not the main use case of a remote). We encapsulate the actual remote code on the `remote-entry` folder so I'd expect that everything related to the remote entry point it's there.

IMHO, it should be the other way around the outer (secondary concern, less useful, maybe not even used in some cases) `app.module.ts` should consume the `entry.routes.ts`. I haven't looked into the standalone components setup for MF, but it doesn't need to match if it doesn't make sense, it's a different setup, but it'd seem to me that if we also have a `remote-entry` folder for standalone setups, there's really no reason to not have an `entry.routes.ts` files there are well instead of the `app.routes.ts`.

Reply:
Changed to do the following:
 - Expose the entry.routes.ts
 - Keep the app.routes.ts, but change it to lazy load the entry.routes.ts for standalone

This will ensure we are only exposing the remote entry stuff via the webpack config, but maintain the consistency across all apps.